.github/workflows/release.yml

```diff
@@ -122,4 +95,28 @@ jobs:
           ${{steps.get-checksums-from-draft-release.outputs.result}}
           EOF
           )
-          
\ No newline at end of file
+          
+      - name: Run Test cases
+        run: |
+          # Setup minikube
+          curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+          install minikube-linux-amd64 /usr/local/bin/minikube
+          minikube start --driver=docker --memory 4096
```

Comment:
I am fine with using Minikube if all the other tools are, but we should probably switch to kind at some point

Reply:
May be we can have separate PR for that.

Reply:
> I am fine with using Minikube if all the other tools are

We actually don't run the tests during a release in other tools.